### PR TITLE
Add visitor & visit count stats endpoint

### PR DESCRIPTION
**Pull request by indrekj**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -1330,3 +1328,44 @@ Possible dimensions that can be used in filters:
...
+        "end_date": "2015-09-03T00:00:00Z",
+        "filter": {}
+      }
+
++ Response 200 (application/json)
+
+    [
+        {
+            "result": {
+                "count": 1898.221896183488752,
```
Comment (deiwin) [inline apiary.apib]: I know this is an approximation, but we should probably round it to an integer. The extra precision provides us nothing.

Comment (indrekj) [inline apiary.apib]: However it yells to user that this is approximation and not an exact value. I remember this is how I noticed that this is an approximation the first time.

Comment (deiwin) [inline apiary.apib]: If we need this to be explicit I'd rather we change the field name to `approximate_count`

**Comment by deiwin** [comment]
The fixup commit hid the precision discussion, but it's still open

**Comment by indrekj** [comment]
https://github.com/salemove/application_services/pull/757

**Comment by deiwin** [comment]
+1 once the *application_services* PR gets merged.

**Comment by indrekj** [comment]
Added visit count documentation to this PR as well to avoid PR conflicts

File: apiary.apib [hunk 2]
```diff
@@ -1322,3 +1320,76 @@ Possible dimensions that can be used in filters:
             "timestamp": "2015-09-02T00:00:00.000Z",
         }
     ]
+
+
+### Visitor Count [POST /visitors/stats/count]
+Returns approximate distinct visitor count. This endpoint is using [HyperLogLog](https://en.wikipedia.org/wiki/HyperLogLog) algorithm to calculate cardinality.
```
Comment (deiwin) [inline apiary.apib]: Maybe describe the approximation in a section and refer to that everywhere where it's used.

Comment (deiwin) [inline apiary.apib]: Ah, the visit count is exact? Never mind then. At least for now

Comment (indrekj) [inline apiary.apib]: I'll do that if there are any other approximate count endpoints. At the moment there's only this one.

Comment (indrekj) [inline apiary.apib]: Yep, it is exact

**Comment by deiwin** [comment]
Ready to be merged

**Comment by deiwin** [comment]
@stenver, !squash :(

**Pull request merged**